scripts/run_xctests.py

```diff
@@ -0,0 +1,201 @@
+from frameworkify import FrameworkBuilder, set_verbosity
+from subprocess import call, Popen, PIPE
+from collections import OrderedDict
+import argparse
+import os
+
+class mini_module:
+    def __init__(self, source_paths, excludes = None, imports = None):
+        self.source_paths = [source_paths]
+        self.excludes = [excludes]
+        self.import_paths = [imports] if imports is not None else None
+
+def run_xctest(fmk_name):
+    exit_code = call(['xcrun', 'xctest', 'Frameworks/macos/{}.xctest'.format(fmk_name)])
```

Comment:
[q] is this going to print test output to stdout/stderr and give us useful logs?

Reply:
Yes, it does. See the evergreen patch for more info